src/dbg/taskthread.cpp

```diff
@@ -0,0 +1,95 @@
+#include <Windows.h>
```

Comment:
Even moving that around I get this kind of thing: https://gist.github.com/jdavidberger/d6b93ffed5195a7ad6139f5d907fc278 in other compilation units. cl is giving a version as Version 18.00.40629 for x86 and VS2013 version 12.0.40629.00 update 5 for what its worth. 

This does bring up though that apparently InitializeConditionVariable isn't available on XP. I'll reimplement TaskThread with a semaphore and fix the header order.